server/routes/index.js

```diff
@@ -1,97 +1,23 @@
-const base58 = require('./base58.js')
```

Comment:
The amount of deleted code here is a great sign, and in the long run it helps make it much, much easier to reason about your code when an error pops up (you'll likely know exactly which file to look in thanks to your stack trace.)

Impressive work! 👏